src/rules/no-standalone-expect.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import { createRule, isDescribe, isExpectCall, isFunction } from './tsUtils';
+
+const getBlockType = (stmt: TSESTree.BlockStatement) => {
+  const func = stmt.parent;
+  // functionDeclaration: function func() {}
+  if (func && func.type === AST_NODE_TYPES.FunctionDeclaration) {
+    return 'function';
+  } else if (func && isFunction(func) && func.parent) {
```

Comment:
could you clean up the else after return here?

Reply:
Merging in master will give lint errors here, btw

Reply:
Not sure what you mean, clean up in what way?

Reply:
Oh should I not pull master?

Reply:
No, please in master, and you'll get lint errors due to https://eslint.org/docs/rules/no-else-return

Reply:
Ah, sorry, I misunderstood. I have master pulled and actually am not seeing errors. I am pretty sure I already removed all of the else returns :)

Reply:
huh, it seems you do! Weird... I fixed them myself, see the commit for what I meant 🙂